### PR TITLE
Inline svg component built

### PR DESCRIPTION
**Pull request by BrianAA**
Created an inline svg component. It takes the property of path where the SVG is located and the component then uses react-svg before-injection to look up all line, rects and paths to override the fill to use our color variables on the site to honor dark mode.

To test go to page [components/icon](https://wpds-docs-git-inline-svg.preview.now.washingtonpost.com/components/icon) and toggle the dark mode the color should change accordingly and verify also that the check list in [visual hidden](https://wpds-docs-git-inline-svg.preview.now.washingtonpost.com/components/visually-hidden) also styling remains the same. 

Some caveats it won't honor static colors as there is no indicator of knowing unless manually flagged in the SVG.

### CODE REVIEW

File: components/Markdown/Components/inlineSVG.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+			c = "0x" + c.join("");
+			let rgba =
+				"rgba(" +
+				[(c >> 16) & 255, (c >> 8) & 255, c & 255].join(",") +
+				",1)";
+			console.log(rgba);
+			return lookUpValue(rgba);
+		}
+		if (hex == "white") {
+			return "var(--wpds-colors-gray600)";
```
Comment (artmsilva) [inline components/Markdown/Components/inlineSVG.js:20]: noice

File: components/Markdown/Components/inlineSVG.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+	function lookUpValue(rgba) {
+		for (var token in Tokens.color.light) {
+			if (Tokens.color.light[token].hasOwnProperty("value")) {
+				const tokenValue = Tokens.color.light[token].value.replaceAll(
+					" ",
+					""
+				);
+				if (rgba == tokenValue) {
+					return `var(--wpds-colors-${token})`;
```
Comment (artmsilva) [inline components/Markdown/Components/inlineSVG.js:37]: oh i see!

File: mdx_components/icon.mdx [hunk 3]
```diff
@@ -7,9 +7,13 @@ component: icon
 ## Anatomy
 
 <Container caption="Icon component shown with an Add icon asset. Inspect the HTML to see the props. Or use a screen reader to read the label.">
-	<EmbedStory componentName="icon" />
+	<InlineSVG path="/img/components/icon/anatomy.svg" />
```
Comment (artmsilva) [inline mdx_components/icon.mdx:10]: Simple

**Review by artmsilva — APPROVED** [review]

File: components/Markdown/Components/list.js [hunk 4]
```diff
@@ -3,10 +3,18 @@ const { styled, theme } = require("@washingtonpost/wpds-ui-kit");
...
+			ol: {
+				listStyle: "revert",
+				paddingLeft: "$100",
+			},
+		},
+	},
 });
 export const ListItem = styled("li", {
 	padding: "$025 0",
+	color: theme.colors.gray80,
```
Comment (artmsilva) [inline components/Markdown/Components/list.js]: i see a potential theme token brewing here! we have 6 instances of "gray80" :)

Comment (BrianAA) [reply components/Markdown/Components/list.js]: oof lol good call out actually gray is the accessible alias token lol  I am going to update those to be accessible alias.

**Pull request merged**